api/src/main/java/org/apache/gravitino/credential/AzureAccountKeyCredential.java

```diff
@@ -0,0 +1,111 @@
+ *  under the License.
+ */
+
+package org.apache.gravitino.credential;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import org.apache.commons.lang3.StringUtils;
+
+/** Azure account key credential. */
+public class AzureAccountKeyCredential implements Credential {
+
+  /** Azure account key credential type. */
+  public static final String AZURE_ACCOUNT_KEY_CREDENTIAL_TYPE = "azure-account-key";
+  /** Azure storage account name */
+  public static final String GRAVITINO_AZURE_STORAGE_ACCOUNT_NAME = "azure-storage-account-name";
+  /** Azure storage account key */
+  public static final String GRAVITINO_AZURE_STORAGE_ACCOUNT_KEY = "azure-storage-account-key";
+
+  private String accountName;
+  private String accountKey;
+
+  /**
+   * Constructs an instance of {@link AzureAccountKeyCredential}.
+   *
+   * @param accountName The Azure account name.
+   * @param accountKey The Azure account key.
+   */
+  public AzureAccountKeyCredential(String accountName, String accountKey) {
+    validate(accountName, accountKey, 0);
+    this.accountName = accountName;
+    this.accountKey = accountKey;
+  }
+
+  /**
+   * This is the constructor that is used by credential factory to create an instance of credential
+   * according to the credential information.
+   */
+  public AzureAccountKeyCredential() {}
+
+  @Override
+  public String credentialType() {
+    return AZURE_ACCOUNT_KEY_CREDENTIAL_TYPE;
+  }
+
+  @Override
+  public long expireTimeInMs() {
+    return 0;
+  }
+
+  @Override
+  public Map<String, String> credentialInfo() {
+    return (new ImmutableMap.Builder<String, String>())
+        .put(GRAVITINO_AZURE_STORAGE_ACCOUNT_NAME, accountName)
+        .put(GRAVITINO_AZURE_STORAGE_ACCOUNT_KEY, accountKey)
+        .build();
+  }
+
+  @Override
+  public void initialize(Map<String, String> credentialInfo, long expireTimeInMS) {
+    String accountName = credentialInfo.get(GRAVITINO_AZURE_STORAGE_ACCOUNT_NAME);
+    String accountKey = credentialInfo.get(GRAVITINO_AZURE_STORAGE_ACCOUNT_KEY);
+    validate(accountName, accountKey, expireTimeInMS);
+    this.accountName = accountName;
+    this.accountKey = accountKey;
+  }
+
+  /**
+   * Get Azure account name
+   *
+   * @return The Azure account name
+   */
+  public String accountName() {
+    return accountName;
+  }
+
+  /**
+   * Get Azure account key
+   *
+   * @return The Azure account key
+   */
+  public String accountKey() {
+    return accountKey;
+  }
+
+  private void validate(String accountName, String accountKey, long expireTimeInMS) {
+    Preconditions.checkArgument(
+        StringUtils.isNotBlank(accountName), "Azure account name should not be empty.");
+    Preconditions.checkArgument(
+        StringUtils.isNotBlank(accountKey), "Azure account key should not be empty.");
+    Preconditions.checkArgument(
+        expireTimeInMS == 0, "The expire time of AzureAccountKeyCredential is not 0");
```

Comment:
This check doesn't make good senses to me.
Could you please double check?

Reply:
Actually can remove this line 
`Preconditions.checkArgument(expireTimeInMS == 0, "The expire time of AzureAccountKeyCredential is not 0");`